SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -313,5 +313,5 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	  * By default, load balancing config will be read from the WorldSettings, but this can be toggled to override
 	  * the map's config with a 1x1 grid.
 	  */
-	bool bOverrideLoadBalancing;
+	bool bOverrideMultiWorker;
```

Comment:
This needs to be an optional bool, that by default isn't set. That way we don't use it when the command line isn't specified.